### PR TITLE
responsive view in mobile devices

### PR DESCRIPTION
**Pull request by rijusougata13**
previous mobile view :
Screenshot (23)

I have used chakra ui flex to make them responsive

### CODE REVIEW

**Comment by rijusougata13** [comment]
PTAL @aquibbaig  @ruddi10

**Comment by ruddi10** [comment]
> PTAL @aquibbaig @ruddi10

@rijusougata13 I think our project is meant for pcs desktops and laptops so it wasn't required to make the screen responsive for mobiles. @aquibbaig what do you think

**Comment by aquibbaig** [comment]
makes sense but I wont discard this change, this certainly is a "good to have".

**Comment by rijusougata13** [comment]
got the point that it's only desktop only so don't need to optimize it for mobile. But still we have to present the dashboard to user such that they can at least use our dashboard on mobile also. So I proposed this small change and Ig we can merge it to the main repo.

**Comment by ruddi10** [comment]
@rijusougata13 yes I also feel we can merge this it's a small change that's good to have 😁

**Comment by ruddi10** [comment]
Hey @rijusougata13 in the ss you have shared I can see white space on the top of bench routes logo please have a look at that, it shouldn't be the case

**Comment by rijusougata13** [comment]
> Hey @rijusougata13 in the ss you have shared I can see white space on the top of bench routes logo please have a look at that, it shouldn't be the case

okay!! on it.

**Review by ruddi10 — CHANGES_REQUESTED** [review]

File: src/pages/Home/index.tsx [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+    >
       <GridItem bg={value} colSpan={5}>
         <Sidebar />
       </GridItem>
-      <GridItem colSpan={19}>
+      <GridItem colSpan={19} flex="1">
         <RouteVisualiser />
       </GridItem>
-    </Grid>
+    </Flex>
```
[the remark this answers is not in the document]
Comment (rijusougata13) [reply src/pages/Home/index.tsx]: Fixed this. PTAL @ruddi10.

**Comment by aquibbaig** [comment]
cc: @ruddi10

**Comment by ruddi10** [comment]
@rijusougata13 you have given Flex props to the Grid I don't think it should work, also even if it's working I don't think it's a good practice.

**Comment by rijusougata13** [comment]
sure @ruddi10  I'll send a fix within tonight removing that

**Comment by rijusougata13** [comment]
Hi removed  the flex @ruddi10. I think its good to go now.

**Review by aquibbaig — APPROVED** [review]

**Comment by ruddi10** [comment]
Hey @rijusougata13 can you just share the screenshot of new desktop and mobile view after these changes then we are good to go

**Comment by rijusougata13** [comment]
image
image

**Comment by rijusougata13** [comment]
@ruddi10

**Comment by ruddi10** [comment]
> @ruddi10 
> 

Hey the height of the error message box on the right don't seem right, and is different from older height. Maybe making the height of the Box 100 vh might solve the problem.

**Comment by ruddi10** [comment]
Also use 100% instead of vh in the Box heights

**Comment by rijusougata13** [comment]
> Also use 100% instead of vh in the Box heights

yep but I have to set the parent height to 100vh to acquire hole height of the page

**Comment by ruddi10** [comment]
@rijusougata13 the desktop view looks fine now can you just share the ss of mobile view once then we are good to go if it looks fine

**Review by muskankhedia — APPROVED** [review]
LGTM!

Suggestion for future: On mobile view, we could show the chart below the filters. A demo design could be:

image

@aquibbaig @ruddi10 @rijusougata13  Please share your thoughts on this.

**Review by ruddi10 — APPROVED** [review]

**Pull request merged**